librz/core/disasm.c

```diff
@@ -2173,6 +2173,18 @@ static void __preline_flag(RzDisasmState *ds, RzFlagItem *flag) {
 	}
 }
 
+// check the equality between names from symbol tab and debug info
+// like sym.func_name and dbg.func_name
+static bool is_sym_dbg_equal(const char *a, const char *b) {
+	if (!rz_str_startswith(a, "sym.") && !rz_str_startswith(a, "dbg.")) {
+		return false;
+	}
+	if (!rz_str_startswith(b, "sym.") && !rz_str_startswith(b, "dbg.")) {
+		return false;
+	}
+	return !strcmp(a + strlen("sym."), b + strlen("sym."));
```

Comment:
not really necessary, but i would change this line into this.
```suggestion
	const size_t sym_n = strlen("sym.");
	return !strcmp(a + sym_n, b + sym_n);
```